letsencrypt-auto

```diff
@@ -14,7 +14,12 @@ VENV_PATH=${VENV_PATH:-"$XDG_DATA_HOME/$VENV_NAME"}
 VENV_BIN=${VENV_PATH}/bin
 
 if test "`id -u`" -ne "0" ; then
-  SUDO=sudo
+  if ! type "sudo" > /dev/null; then
+    su
```

Comment:
Mmmm... Didn't expect that one to be that much trouble ._.

According to that link, http://stackoverflow.com/a/17758312/1256113, if I'd like this to work then I'll have to save the whole following block as a string and then execute it. I tried using ```"su [user] -c "[command here]"``` and it asked for the password every single time, which isn't really favourable, I assume.

What are your thoughts?